Sources/MountebankSwift/Documentation.docc/Tutorials/uitesting-refresher.tutorial

```diff
@@ -0,0 +1,33 @@
+@Tutorial {
```

Comment:
This tutorial is empty. Could you remove it? So we can merge the other parts of the pr?